crashlog/crashutils.c

```diff
@@ -178,6 +180,30 @@ void do_last_kmsg_copy(char *dir) {
         snprintf(destion, sizeof(destion), "%s/%s", dir, FTRACE_RAMOOPS_FILE);
         do_copy_tail(FTRACE_RAMOOPS, destion, MAXFILESIZE);
     }
+    if (!find_system_last_kmsg(source)) {
+        snprintf(destion, sizeof(destion), "%s/%s", dir, source);
+        snprintf(sourcepath, sizeof(sourcepath), "%s/%s", DROPBOX_DIR, source);
+        do_copy_tail(sourcepath, destion, MAXFILESIZE);
+    }
+}
+
+// Find system last kmsg from dropbox
+int find_system_last_kmsg(char *source) {
```

Comment:
1, Can we declare this function as static? 2, Should we check if source is NULL?